website/docs/src/css/custom.css

```
@@ -117,13 +112,30 @@
+  color: rgba(255, 255, 255, 0.82);
+}
+
 .navbar__items button[class*='colorModeToggle']:hover {
   color: #ffffff;
-  background: rgba(255, 255, 255, 0.08);
+  background: rgba(255, 255, 255, 0.16);
+}
+
+.navbar__items button[class*='colorModeToggle'] svg,
+.navbar__items button[class*='colorModeToggle'] path {
+  fill: currentColor;
```

Stylelint (17.4.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)